src/packages/blits/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import Badge, { BadgeProps } from './Badge.ts';
+import { Meta, StoryObj } from '@storybook/html';
+import lightning from '../../assets/images/ic_lightning_white_32.png';
+
+// More on how to set up stories at: https://storybook.js.org/docs/7.0/solid/writing-stories/introduction
```

Comment:
Can we update these comment links? It looks like they've been updated and redirect to https://storybook.js.org/docs/writing-stories/args

Reply:
Thanks David good spot, I've updated these for all the stories to go to the new links